src/command_modules/azure-cli-interactive/azclishell/threads.py

```diff
@@ -0,0 +1,45 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import threading
+
+
+class ContinuousPingThread(threading.Thread):
```

Comment:
What is this for? The class itself doesn't explain the usage of it. The sleep 25 seconds for each loop and 4 seconds before quit is too arbitrary. 
1. Can this class be replaced by simply creating a `Thread` using a method?
2. How reusable the logic it is to dedicate a class?
3. Why 4, and 25 seconds respectively. Why aren't they configurable?